### PR TITLE
Centralize location set handling

### PR DESCRIPTION
**Pull request by collijk**

### CODE REVIEW

**Review by miker985 — APPROVED** [review]

File: src/covid_model_deaths/deaths_io/__init__.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 from .inputs import MEASURES, InputsContext
 from .checkpoint import Checkpoint
+from .loctions import get_locations, LOCATION_GROUPS
```
Comment (JosephWagner) [inline src/covid_model_deaths/deaths_io/__init__.py:3]: module is misnamed

Comment (JosephWagner) [reply src/covid_model_deaths/deaths_io/__init__.py:3]: I mean it's correct, but it's not what you want

File: src/covid_model_deaths/deaths_io/loctions.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+LOCATION_GROUPS = _LocationGroups()
+
+
+def get_locations(location_group: str, location_set_version_id: int) -> pd.DataFrame:
+    loc_df = get_location_metadata(location_set_id=111,
+                                   location_set_version_id=location_set_version_id)
+    most_detailed = loc_df['most_detailed'] == 1
+    good_ids = ~loc_df['location_id'].isin([53474, 53451, 53452])
```
Comment (JosephWagner) [inline src/covid_model_deaths/deaths_io/loctions.py:19]: what is going on here? I don't have enough context to know what these 'bad' ids represent

File: src/covid_model_deaths/deaths_io/loctions.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    keep_columns = ['location_id', 'location_ascii_name', 'parent_id', 'level', 'most_detailed']
+
+    loc_df = loc_df.loc[most_detailed * good_ids, keep_columns]
```
Comment (JosephWagner) [inline src/covid_model_deaths/deaths_io/loctions.py:22]: didn't know about this * syntax. Nice.

**Review by JosephWagner — CHANGES_REQUESTED** [review]
Besides module name, lgtm

File: src/covid_model_deaths/deaths_io/loctions.py (hunk 3 above)
Comment (collijk) [reply src/covid_model_deaths/deaths_io/loctions.py:22]: Hah, I think this is an error that happens to still do what I want.  Bitwise & and vectorized * work the same on boolean masks.  Not intentional though.

**Pull request closed without merging**